scripts/deposit-into-mocktoken-strategy.sh

```diff
@@ -14,4 +14,4 @@ CHAINID=$(cast chain-id)
 DEPLOYMENT_OUTPUT_FILE=./contracts/script/output/${CHAINID}/credible_squaring_avs_deployment_output.json
 STRATEGY_ADDRESS=$(jq -r '.addresses.erc20MockStrategy' $DEPLOYMENT_OUTPUT_FILE)
 
-go run cli/main.go --config config-files/operator.anvil.yaml deposit-into-strategy --strategy-addr ${STRATEGY_ADDRESS} --amount 100
+go run cli/main.go --config config-files/${CHAINID}/operator.anvil.yaml deposit-into-strategy --strategy-addr ${STRATEGY_ADDRESS} --amount 100
```

Comment:
@wawrzek could you update this to reflect 2 items:
1. to use `${CHAINID:?}` (in spite of `make CHAIND=32382...` working)
2. update the README to highlight that `start-aggregator` should be run with a CHAINID env var

Otherwise lgtm

Reply:
@NimaVaziri , I've been looking at changing the behaviour of the make commands and I don't like it. Despite me wanting to use a different chain id I think the change should not make life harder for others, so keep the current behaviour when the CHAINID might be omitted.
I suggest an alternative. Keep current Makefile, but extend documentation by a paragraph describing how to deploy AVS into a network different from the anvil one. The change is in the latest commit.